optaplanner-core/src/main/java/org/optaplanner/core/api/score/buildin/bendable/BendableScore.java

```diff
@@ -149,18 +149,6 @@ public int getLevelsSize() {
         return hardScores.length + softScores.length;
     }
 
-    /**
-     * @param level {@code 0 <= level <} {@link #getLevelsSize()}
-     * @return higher is better
-     */
-    public int getHardOrSoftScore(int level) {
```

Comment:
I didn't see any usages in the project (true, there can be clients who actually use it). I'll revert the change.